awx/main/scheduler/task_manager.py

```diff
@@ -552,22 +552,23 @@ def start_task(self, task, instance_group, dependent_tasks=None, instance=None):
                 task.save()
                 task.log_lifecycle("waiting")
 
-        def post_commit():
-            if task.status != 'failed' and type(task) is not WorkflowJob:
-                # Before task is dispatched, ensure that job_event partitions exist
-                create_partition(task.event_class._meta.db_table, start=task.created)
```

Comment:
@fosterseth here `create_partition` is moved into the task code when the job starts. This, plus removing the "waiting" websocket message, are both changes that remove `on_commit` logic from the task manager.